gen/dcos-config.yaml

```diff
@@ -109,17 +182,44 @@ package:
   - path: /etc/mesos-master-provider
     content: |
       MESOS_CLUSTER={{ cluster_name }}
-  - path: /etc/mesos-slave-modules.json
+  {% switch dcos_overlay_enable %}
+  {% case "true" %}
+  - path: /etc/mesos-slave-modules/overlay_slave_modules.json
```

Comment:
Downstream dcos's will likely add their own additional modules, and it may
make sense to keep all the agent modules together?

On Tue, Jun 28, 2016 at 11:19 PM, Sargun Dhillon <notifications@github.com>
wrote:

> In gen/dcos-config.yaml
> <https://github.com/dcos/dcos/pull/282#discussion_r68890625>:
>
> > @@ -109,17 +182,44 @@ package:
> >    - path: /etc/mesos-master-provider
> >      content: |
> >        MESOS_CLUSTER={{ cluster_name }}
> > -  - path: /etc/mesos-slave-modules.json
> > +  {% switch dcos_overlay_enable %}
> > +  {% case "true" %}
> > +  - path: /etc/mesos-slave-modules/overlay_slave_modules.json
>
> Any reason not to put both mesos-master-modules, and mesos-slave modules
> config in the same switch statement?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dcos/dcos/pull/282/files/8dea7b12f03f3a0fcb20374f7afdb7b57d73ae71#r68890625>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGKE0HTqlFrVOZkl4CANSbWR654-q5VQks5qQg55gaJpZM4I_eCx>
> .
>